test/smoke/src/areas/terminal/terminal-editors.test.ts

```diff
@@ -0,0 +1,80 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ParsedArgs } from 'minimist';
+import { Code, Terminal, TerminalCommandId, TerminalCommandIdWithValue } from '../../../../automation/out';
+import { afterSuite, beforeSuite } from '../../utils';
+
+const SPLIT_BUTTON_SELECTOR = '.editor .codicon-split-horizontal';
```

Comment:
`terminal.clickEditorSplitButton`?

Reply:
wasn't sure if we should add that because it's only used once

Reply:
move to terminal.ts with all other selectors and make Selector. enum